sDownloads.ps1

```diff
@@ -0,0 +1,27 @@
+# Check directory sDownloads.ps1 runs from for sd-config file
+$ConfigPath = $PSScriptRoot + "\sd-config"
+
+# sd-config needs to stay with sDownloads.ps1
+# Change the path if you want sd-config somewhere else
+$Config = Import-Csv -Path $ConfigPath
+
+# Will work fine as long as you use the default Downloads folder
+$Downloads = "C:\Users\$env:UserName\Downloads\"
+
+$Files = $Downloads | Get-ChildItem
+$FileNames = $Files.Name
```

Comment:
I think these two lines could be merged into one. It may make it more readable. 
Alternatively, you may not want to be collecting only the names, and instead just stick with $Files and the whole object as I'll mention below.